tests/test_wave_05.py

```diff
@@ -52,12 +52,19 @@ def test_new_genre_rec_from_empty_friends():
             }
         ]
     }
+    # Act
+    recommendations = get_new_rec_by_genre(sonyas_data)
+
+    # Assert
+    for rec in recommendations:
+        assert rec['genre'] == 'Intrigue'
+    assert len(recommendations) == 0
```

Comment:
These assertions are kind of contradictory since the `for` loop should never be entered if the list is empty.

```suggestion
    assert len(recommendations) == 0
```